packages/ui/src/kit/VTable/VTableHeadCell/VTableHeadCell.vue

```diff
@@ -1,21 +1,97 @@
 <template>
   <th :class="vTableHeadCellCssClass">
-    <slot />
+    <a
+      href="#"
+      class="v-table-head-cell-sort-wrapper"
+      @click.prevent="onClick"
+    >
+      <slot />
+      <svg-atom
+        v-if="isSortable"
+        class="v-table-head-cell-sort-icon"
+        :name="sortIcon"
+        width="8"
+        height="8"
+      />
+    </a>
   </th>
 </template>
 
 <script>
 import ThemeMixin from '@/mixins/ThemeMixin';
+import SvgAtom from '@/atom/svg-atom/svg-atom';
+
+const SORTING_DIRECTIONS = {
+  DESC: 'desc',
+  ASC: 'asc',
+};
 
 export default {
   name: 'VTableHeadCell',
+
+  props: {
+    sortingBy: {
```

Comment:
no need to pass `sortingBy`, it's enought to pass only sort direction

Reply:
you aren't right
we anyway should pass some flag, and this is not direction
so you just suggest to replace `sortingBy` to `isSortable`
also if cell knows about sort direction, cell anyways know about direction and responsible for that, so no crime to let it know about their sorting value just to determine that current cell is sortable

for now I will pass `isSortable` flag and this change will make using sortable table more difficult

Reply:
direction can be in those values:
- no direction (default state)
- asc
- desc

and when you click to head cell, you switching only this values and emit event about new direction